cohort-engine/src/main/java/com/ibm/cohort/engine/measure/CDMMeasureEvaluation.java

```diff
@@ -195,7 +197,31 @@ protected static void addDefineEvaluationToReport(MeasureReport report, CDMConte
 			}
 		}
 	}
-	
+
+	/**
+	 * Set the measure reference on a MeasureReport to the id of a given Measure.
+	 * Attempt to normalize the id to just the portion after "Measure/" if possible
+	 *
+	 * Examples:
+	 *   Measure/id1234
+	 *   Measure/id1234/_history/10
+	 *
+	 * If the id does not contain "Measure/", then the report measure is set to the
+	 * full id of the measure.
+	 *
+	 * @param report MeasureReport on which to set the measure reference
+	 * @param measure Measure providing the id to normalize and set on the report
+	 */
+	protected static void setReportMeasureToMeasureId(MeasureReport report, Measure measure) {
```

Comment:
```java
String measureId1 = "Measure/test";
String measureId2 = "hello/Measure/test";
String measureId3 = "test";
String measureIdPrefix = "^(Measure/)";
String measure1 = measureId1.replaceAll(measureIdPrefix, "");
String measure2 = measureId2.replaceAll(measureIdPrefix, "");
String measure3 = measureId3.replaceAll(measureIdPrefix, "");

System.out.println(measureId1 + " -> " + measure1);
System.out.println(measureId2 + " -> " + measure2);
System.out.println(measureId3 + " -> " + measure3);

// Measure/test -> test
// hello/Measure/test -> hello/Measure/test
// test -> test 
```

It might be a bit cleaner if I understand the requirement correctly:
mainly because I don't like `if`s 😛 

Not a blocker, though.

Reply:
Looking over your unit test again, I think I may have misunderstood what you were trying to do with `startOfId` and `substring`, so I think you can ignore this comment completely.

Reply:
Is there anything code-wise or comment-wise that you think  could clear things up a bit?

In your examples:
* `Measure/test` stays `Measure/test`
* `hello/Measure/test` becomes `Measure/test`
* `test` stays `test`

Reply:
Maybe the javadoc could give some before/after conversions?